lib/stdlib/src/parse_nginx_log.rs

```diff
@@ -168,6 +175,34 @@ fn kind_combined() -> BTreeMap<Field, Kind> {
     ])
 }
 
+fn kind_ingress_upstreaminfo() -> BTreeMap<Field, Kind> {
+    BTreeMap::from([
+        ("remote_addr".into(), Kind::bytes()),
+        ("remote_user".into(), Kind::bytes().or_null()),
```

Comment:
null means that the key actually exists, and has a null value. In this case, the remote_user key might not exist at all, so it should be undefined instead. Same for all optional fields here.
```suggestion
        ("remote_user".into(), Kind::bytes().or_undefined()),
```